weather-app/src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1111111,
+    description: "Stub description",
+    icon: 800,
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+  };
```

Comment:
You are missing a required prop here:
```
      max: 22,
    },
    onSelect: jest.fn(),
  };
```
This will then have to be added to all the instantiations of the component below